.gitignore

```diff
@@ -331,7 +331,7 @@ BuildOutput/
 installer/dev/project_reunion_definitions_override.h
 
 # Installer test packages
-!installer/test/testpackages/
+installer/test/testpackages/
```

Comment:
It is being ignored because that's where the packages are placed by the makefile to work around resource compiler limitations in macro processing. That can be changed when it moves over to data-file input for packages.